### PR TITLE
Added the Hypothesis development setup doc

### PR DESCRIPTION
**Pull request by mik-dass**
Signed-off-by: mik-dass mrinald7@gmail.com

### CODE REVIEW

File: HypothesisSetup.asciidoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+1. Fork this repo: https://github.com/fabric8-hdd/fabric8-hypothesis-core-image.git
+2. To change the default node version, npm version, push registry, pull registry, organisation and port number, change the respective values in the constants.sh 
+3. Trigger the build using cico_build_deploy.sh script and wait for pushes to be complete.
+ 
+
+# Building the fabric8-hypothesis-manager-api-image image
+
+1. First make sure that you have built and pushed the fabric8-hypothesis-core-image image with the approriate default node version, npm version, push registry, pull registry, organisation and port values.
+2. Fork this repo: https://github.com/fabric8-hdd/fabric8-hypothesis-manager-api-image.git
+* For changing the set_env.sh file location of fabric8-hypothesis-core-image, change the env_script_location value in the app_constants.sh file.
```
Comment (anmolbabu) [inline HypothesisSetup.asciidoc]: Not required

File: HypothesisSetup.asciidoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* For changing the app_name, change the APP_NAME value in the app_constants.sh file.
```
Comment (anmolbabu) [inline HypothesisSetup.asciidoc]: try explaining the imapact of changing this like if you change app_name image name will be changed, etc...

File: HypothesisSetup.asciidoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* For changing the port, route_path, route_prefix and hostname, change the respective values in the supervisor.env file in the openshift folder.
```
Comment (anmolbabu) [inline HypothesisSetup.asciidoc]: please put a note that the changes to supervisor.env will be affected only for deployments using deploy.sh and will not be applicable for the default osio CI/CD, the port as a parameter needs to be changed at the time of image build as well as the port exposed for service will be exposed at image build time itself.

File: HypothesisSetup.asciidoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+2. Fork this repo: https://github.com/fabric8-hdd/fabric8-hypothesis-data-api-image.git
+* For changing the set_env.sh file location of fabric8-hypothesis-core-image, change the env_script_location value in the app_constants.sh file.
+* For changing the app_name, change the APP_NAME value in the app_constants.sh file.
+* For changing the port, route_path, route_prefix and hostname, change the respective values in the data-api.env file in the openshift folder.
+3. Trigger the build using cico_build_deploy.sh script and wait for pushes to be complete.
+
+
+# Building the fabric8-hypothesis-measurements-api-image image
+
+1. First make sure that you have built and pushed the fabric8-hypothesis-core-image image with the approriate default node version, npm version, push registry, pull registry, organisation and port values.
```
Comment (anmolbabu) [inline HypothesisSetup.asciidoc]: there's no image for hypothesis-core-image but instead the image built from hypothesis-core-image is nodejs image which is a base layer for other images

**Review by anmolbabu — CHANGES_REQUESTED** [review]

File: HypothesisSetup.asciidoc (hunk 1 above)
Comment (mik-dass) [reply HypothesisSetup.asciidoc]: Even the fabric8-hypothesis-core-image part?

Comment (anmolbabu) [reply HypothesisSetup.asciidoc]: env_script_location changes should not be suggested ideally

**Comment by anmolbabu** [comment]
LGTM
But can you also try explaining how to deploy the same on a local minishift cluster

**Review by anmolbabu — APPROVED** [review]

**Pull request merged**